win32ss/user/ntuser/menu.c

```diff
@@ -3001,8 +3001,8 @@ static BOOL FASTCALL MENU_ShowPopup(PWND pwndOwner, PMENU menu, UINT id, UINT fl
     /* We are off the left side of the screen */
     if (x < monitor->rcMonitor.left)
     {
-        /* Re-orient the menu around the x-axis */
-        x += width;
+        /* Position menu at left edge of screen */
+        x = 0;
```

Comment:
* Could you explain briefly why `0` is special, especially compared to `monitor->rcMonitor.left`?
* If you change this code (in any way), should you change line 3014 in a similar way?

Reply:
In the linked JIRA report [CORE-16729](https://jira.reactos.org/browse/CORE-16729), there is this image:
https://jira.reactos.org/secure/attachment/56195/Bad%20Menu.png
that I annotate below for our understanding:
![image](https://github.com/reactos/reactos/assets/1969829/57de7ecc-36f3-4884-a15b-6a36f102ad87)

Doing this, I see that setting x to zero may be questionable. @Doug-Lyons could you recheck this?

Reply:
@HBelusca and @SergeGautherie, In ReactOS' present state ```monitor->rcMonitor.left``` is exactly equal to ```0```.
With the zero (0) being shorter and cleaner, I elected to go with it here. I did give considerable thought to this approach.
I went to the effort to do a Jira search to find the current state of ReactOS' multi-monitor support and it is essentially non-existent at this time. After this is introduced, this code will need considerable attention to see how it integrates with the current menu position handling. Until that time, there is really nothing better that can be done here than to use the zero (0) as I have already done. Thanks.

Reply:
> * Could you explain briefly why `0` is special, especially compared to `monitor->rcMonitor.left`?
> * If you change this code (in any way), should you change line 3014 in a similar way?

@SergeGautherie, There is not any need at this time to modify line 3014. If you try and move the menu bar out of the visible screen area toward the top of the screen, our automatic ```snap``` will always move the window back down into the visible window area before you can re-position the "menu" even partially off-screen. Thanks.

Reply:
> In ReactOS' present state monitor->rcMonitor.left is exactly equal to 0.

I would say, forget about that fact and code as if rcMonitor.left could be something else; because otherwise someone would have to find again all these instances and potentially fix them when we fully support multi-monitor setups (whose implementation has been started by @hpoussin)